README.md

```diff
@@ -40,7 +40,16 @@ jsonfile.readFile(file, function(err, obj) {
   console.dir(obj)
 })
 ```
+### readFile(filename, [options]) : Promise
```

Comment:
Rather than adding a new method heading, just add a note to the item above stating that promises are supported, and will be returned if no callback is passed.